utils/build-presets.ini

```diff
@@ -1701,6 +1701,26 @@ assertions
 swiftsyntax
 swiftsyntax-verify-generated-files
 
+#===------------------------------------------------------------------------===#
+# Test Swift Format
+#===------------------------------------------------------------------------===#
+
+[preset: buildbot_swiftformat_macos]
+mixin-preset=mixin_swiftpm_package_macos_platform
+release
+assertions
+swiftsyntax
+swiftsyntax-verify-generated-files
```

Comment:
I think this is only useful for `swift` / `swift-syntax` testing. (/cc @ahoppen).

Reply:
The comments in this file suggest this is used to verify the checked-in generated files are correct, since they won't be generated. I guess it's safe to assume that the checked-in generated files are safe to use in this context without verifying them?

Reply:
>  I guess it's safe to assume that the checked-in generated files are safe to use in this context without verifying them?

I believe so.

Reply:
Sounds good. I have deleted this step, and added `skip-test-swiftsyntax` to also skip running swiftsyntax tests.